tests/test_doc_auto_generation.py

```diff
@@ -0,0 +1,78 @@
+from docs import autogen
+import pytest
+test_doc1 = {
+    'doc': '''This function does something really interesting.
```

Comment:
Prefer using a real docstring, such as `ImageDataGenerator.__doc__`

Reply:
Hi @fchollet , thank you for your feedback.
As regards this point, I am not quite sure about it. I do not like the idea of having a test that breaks whenever someone changes the doc (i.e fixing a typo). I think this should rather be seen as a contract (aka - how to write a doc in order to obtain something correctly rendered) rather than something physically connected with a particular doc string. This allows also to explore all the potential functionalities.
What do you think?
I suggest to move the test string in an external file that can be loaded, but I am opened to any suggestion.